tests/test_utils.py

```diff
@@ -79,6 +79,124 @@ def test_concat(objects, axis):
         )
 
 
+@pytest.mark.parametrize(
+    'objs, expected',
+    [
+        (
+            [],
+            audformat.filewise_index(),
+        ),
+        (
+            [audformat.filewise_index()],
+            audformat.filewise_index(),
+        ),
```

Comment:
Could you add in addition an example containing:

```python
[
    audformat.filewise_index(),
    audformat.filewise_index(),
]
```

Reply:
I added it and also one with `segmented_index()`.